lib/librender.c

```diff
@@ -465,58 +479,83 @@ void free4DTensor(float ****tensor, int dim1, int dim2, int dim3){
 	free(tensor);
 }
 
-double*** init3DTensor(int dim1, int dim2, int dim3, double init){
-    
+double*** init3DTensor(int dim1, int dim2, int dim3, double init)
+{
     double ***tensor = (double ***)malloc(dim1*sizeof(double**));
     
     if(tensor == NULL){
-        fprintf(stderr,"failed to initialize 3D tensor (1)");
+        vol2bird_err_printf("failed to initialize 3D tensor (1)");
+#ifdef VOL2BIRD_R
```

Comment:
Since this standalone vol2bird, we don't have to worry about setting VOL2BIRD_R flag, correct?